db/migrate/20240619101531_create_content_blocks.rb

```diff
@@ -0,0 +1,10 @@
+class CreateContentBlocks < ActiveRecord::Migration[7.1]
```

Comment:
I think we'll need to inherit from the Edition model and use the `editions` table here. There's some prior art in this PR - https://github.com/alphagov/whitehall/pull/9112